### ISSUE TITLE
Assignment 11-Q4-d

### ISSUE THREAD

**Issue by John-BME** [BME1478H/Fall2020class#271]
Hi, 
what is the difference between score for training data and test data and is the correlation between the actual and predicted life expectancy values defined by score in test data as follows:
corr_tree = tree_model.score(X_test, Y_test)

**Comment by joelostblom** [comment]
Generally, you should not score your model on the data you have trained it with because there is a risk of it only being good at that data (overfitting). If you score it on data it has not seen during training, you get a sense for how well the model generalizes (which often is a desirable property).

The [score is the mean accuracy on the test data.](https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html#sklearn.tree.DecisionTreeClassifier.score)
